build/make/lz4defs.make

```diff
@@ -88,6 +88,11 @@ else
 POSIX_ENV = No
 endif
 
+# `AOCL_PRELOAD_BRANCH_OPT` Enable data preload for byte matching within the inner loop in 
+# LZ4_compress_generic_validated and better branch predictor behavior for noDict directive.
+
+
+USEROPTIONALFLAGS = -DAOCL_PRELOAD_BRANCH_OPT=1
```

Comment:
nit: I have a preference for :

```
USEROPTIONALFLAGS ?= -DAOCL_PRELOAD_BRANCH_OPT=1
```

so that `USEROPTIONALFLAGS` can also be set via environment variable (on top of command line argument, which remains higher priority).